index.html

```diff
@@ -0,0 +1,1117 @@
+              </path>
+            </g>
+          </svg>
+          <div class="mt-4 copyright">© 2020 NEWSWEEK</div>
+        </div>
+        <div class="col-md-2 social-icons d-flex justify-content-around mt-4 p-0">
+          <i class="fab fa-facebook-square fa-lg"></i>
+          <i class="fab  fa-twitter fa-lg"></i>
+          <i class="fab fa-instagram fa-lg"></i>
+          <i class="fab fa-tumblr fa-lg"></i>
+          <i class="fab fa-linkedin-in fa-lg"></i>
+        </div>
+      </div>
+      <div class="col-md-5 footer-list p-0">
+        <ul class="d-flex justify-content-between p-0">
+          <li><b>Editions:</b></li>
+          <li class="">U.S. Edition</li>
+          <li class="border-left pl-4">日本</li>
+          <li class="border-left pl-4 ml-3">Pakistan</li>
+          <li class="border-left ml-4 pl-3">Polska</li>
+          <li class="ml-3">România</li>
+        </ul>
+      </div>
+      <div class="col-md-12 footer-list p-0 pb-">
+        <ul class="d-flex justify-content-between flex-wrap p-0">
+          <li class="border-right text-center pr-3 ml-1">About Us</li>
+          <li class="border-right text-center pr-3 ml-1"> Corrections</li>
+          <li class="border-right text-center pr-3 ml-1">Contact Us</li>
+          <li class="border-right text-center pr-3 ml-1">Editorial Guidelines</li>
+          <li class="border-right text-center pr-3 ml-1">Advertise</li>
+          <li class="border-right text-center pr-3 ml-1">Copyright</li>
+          <li class="border-right text-center pr-3 ml-1">Terms & Conditions</li>
+          <li class="border-right text-center pr-3 ml-1">Privacy Policy</li>
+          <li class="border-right text-center pr-3 ml-1">Cookie Policy</li>
+          <li class="border-right text-center pr-3 ml-1">Terms of Sale</li>
+          <li class="border-right text-center ml-1 pr-3">Archive</li>
+          <li class="border-right text-center pr-3 ml-1">Announcements</li>
+          <li class="ml-1 text-center">Consent preferences</li>
```

Comment:
Check the font color of these elements and make them look like the original page.
When you are on medium and mobile sizes., your footer elements should be aligned left.